packages/react/avatar/src/Avatar.stories.tsx

```diff
@@ -33,6 +44,31 @@ export const Styled = () => (
         <AvatarIcon />
       </Avatar.Fallback>
     </Avatar.Root>
+
+    <h1>With image framework component</h1>
+    <Avatar.Root className={rootClass()}>
+      <Avatar.Image
+        className={imageClass()}
+        alt="John Smith"
+        onLoadingStatusChange={console.log}
+        asChild
+      >
+        <FakeFrameworkImage src={otherSrc} />
+      </Avatar.Image>
+      <Avatar.Fallback className={fallbackClass()}>
+        <AvatarIcon />
+      </Avatar.Fallback>
+    </Avatar.Root>
+
+    <h1>With image framework component & with fallback (but broken src)</h1>
+    <Avatar.Root className={rootClass()}>
+      <Avatar.Image className={imageClass()} alt="John Smith" onLoadingStatusChange={console.log}>
```

Comment:
```suggestion
      <Avatar.Image className={imageClass()} alt="John Smith" onLoadingStatusChange={console.log} asChild>
```

if `asChild` is not passed it will still render default `Primitive.img`